### ISSUE TITLE
A11y_SQL Azure Data Studio_Import ExtensionModify Columns_AI4D: The primary key check box in column 3 has no name set.

### ISSUE THREAD

**Issue by Roopesh-T** [microsoft/azuredatastudio#22372]
“[Check out Accessibility Insights! ](https://accessibilityinsights.io/)– Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags**: #A11yMAS;#A11yTCS;#A11ySev2;#BM_SQLAzureDataStudio_Oct2020;#SQL Azure Data Studio;#Win11;#FTP;#DesktopApp;#WCAG4.1.2;#Benchmark;#AI4D;#AILimited;

## Environment Details:
Application Name: SQL Azure Data Studio insider
Version: 1.43.0-insider (user setup)
Commit: 9d16a48dee06c0533c5c6e405e775685d098a5ae
Date: 2023-03-20T05:49:59.560Z
Windows Version: Windows11

## Repro Steps:

Launch Azure Data Studio Insiders application. install "SQL Server Import" extension.
Tab to server drop down and tab again to button new connection press enter key and add the server details.
Tab to the connected server and press enter key, use down key and select any database keep_MDW Right click on/ open the context menu for a database and select Import wizard from the context menu.
upload dacpac file.
Add data in specify input file screen and tab to "next" button and press enter.
Press enter on next button.
Launch accessibility insights for desktop and hover over checkbox in column 3,4 row 2
## Actual:
On hovering over the checkbox in column 3,4 row 2 there is no name set for it.

## Expected:
The name should be set for all controls according to its use.

## User Impact:
The screen reader users will not get the right reference for that control on what it is used for.

## Attachment:
![22372_A11y_SQL Azure Data Studio_Import ExtensionModify Columns_AI4D 1](https://user-images.githubusercontent.com/86601127/226327640-7dadde44-83de-42ce-bf22-23538cf478d6.png)
![22372_A11y_SQL Azure Data Studio_Import ExtensionModify Columns_AI4D](https://user-images.githubusercontent.com/86601127/226327654-8aaec42d-6b0e-43ab-a559-dd871623bee8.png)

**Comment by aasimkhan30** [comment]
@Roopesh-T , the fix is already made in the product. Can you please retest and close?

**Comment by Roopesh-T** [comment]
@aasimkhan30 the issue is still happening in the 
Version: 1.46.0-insider (user setup)
Commit: 321117014dbd9a7a93208ec65253587b2b923042
Date: 2023-07-26T05:36:50.137Z
VS Code: 1.79.0
Electron: 22.3.14
Chromium: 108.0.5359.215
Node.js: 16.17.1
V8: 10.8.168.25-electron.0
OS: Windows_NT x64 10.0.22621


can you please let me know in which version the issue is fixed?
![image](https://github.com/microsoft/azuredatastudio/assets/86601127/44d07315-68af-41cd-8c3a-53c36f7ac6ce)

**Comment by Roopesh-T** [comment]
closing the bug as fixed 
![image](https://github.com/microsoft/azuredatastudio/assets/86601127/16d3df38-f716-4748-bc60-4795518d91d5)
![image](https://github.com/microsoft/azuredatastudio/assets/86601127/7c7d791b-17d4-4ad8-9e9f-3dce17c96d3b)
Version: 1.47.0-insider (user setup)
Commit: de8fbc5254b9eac3d68a4d79c781939ddfa7f4f0
Date: 2023-10-13T05:40:53.886Z
VS Code: 1.79.2
Electron: 22.3.25
Chromium: 108.0.5359.215
Node.js: 16.17.1
V8: 10.8.168.25-electron.0
OS: Windows_NT x64 10.0.22621
